lib/utils/analytics.ts

```diff
@@ -19,23 +19,45 @@ function initiateAnalytics() {
  * @param {string} property - The property of the event
  * @param {string} value - The value of the event
  */
-function useAnalytics() {
-  const { user } = useSupabaseAuth();
+async function captureAnalytics({ title, property, value }: AnalyticEvent) {
+  const analyticsObject: Record<string, string> = {};
```

Comment:
I put this back to a normal function. Having this as a hook was causing it to fire more than once.